integrationtest/src/test/java/org/hibernate/ogm/test/integration/cassandra/ModulesMagicDeckIT.java

```diff
@@ -44,7 +44,7 @@
 				.addClasses( MagicCard.class, MagicCardsCollectionBean.class, ModulesMagicDeckIT.class, MagiccardsDatabaseScenario.class );
 		String persistenceXml = persistenceXml().exportAsString();
 		webArchive.addAsResource( new StringAsset( persistenceXml ), "META-INF/persistence.xml" );
-		ModulesHelper.addModulesDependencyDeclaration( webArchive, "org.hibernate.ogm services, org.hibernate.ogm.cassandra services" );
+		ModulesHelper.addModulesDependencyDeclaration( webArchive, "org.hibernate.ogm:${hibernate-ogm.module.slot} services, org.hibernate.ogm.cassandra:${hibernate-ogm.module.slot} services" );
```

Comment:
I suppose that's going to be replaced via Maven resource filtering? Let's give it a go, though my experiences with it in IDEs are mixed (every now and then, the unfiltered value would show up in the classes dir). But it might be more practicable nowadays, so +1.